.gitignore

```diff
@@ -5,7 +5,7 @@
 .DS_Store
 .*swp
 *.log
-Book/_book
```

Comment:
I guess I should put this line back.
And remove the files that were commited which belong to this folder